evaluate_tflite.py

```diff
@@ -29,9 +29,11 @@
 Then run with a TF Lite model, a scorer and a CSV test file
 '''
 
-def tflite_worker(model, scorer, queue_in, queue_out, gpu_mask):
+def tflite_worker(model, scorer, beam_width, queue_in, queue_out, gpu_mask):
     os.environ['CUDA_VISIBLE_DEVICES'] = str(gpu_mask)
     ds = Model(model)
+    ds.enableExternalScorer(scorer)
+    ds.setBeamWidth(beam_width)
```

Comment:
Beam width is stored in the model file, we want to keep `evaluate_tflite.py` as close as possible to use default values from models.

Reply:
It makes it even simpler then. BeamWidth removed.